### PR TITLE
Remove broken Link

### PR DESCRIPTION
**Pull request by adonno**
Seems that the CLA link to github is not existing anymore. I have been looking to find a replacement but this is all I found 
https://github.com/github/hub/issues/1781. So I thought I might Remove the Link

**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please rebase to the `current` branch

File: CLA.md [hunk 1]
```diff
@@ -38,10 +38,6 @@ is covered under an appropriate open source license and you have the right under
 whether created in whole or in part by you, or you have clearly identified the source of the contribution and any license or other
 restriction (like related patents, trademarks, and license agreements) of which you are personally aware.
 
-## Attribution
-
-This Contributor License Agreement is adapted from the [GitHub CLA][github-cla].
```
Comment (frenck) [inline CLA.md:43]: Let's not remove the full attribution, instead, let's just remove the link (please note, the link is referenced in the bottom of the file as well.)

**Comment by frenck** [comment]
As requested on Discord, rebased the PR onto the `current` branch.

**Review by frenck — APPROVED** [review]
Thanks, @adonno! 👍 

Happy Hacktoberfest 🎉

**Pull request merged**